raiden-ts/tests/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+	"rules": {
+		"import/order": "off",
+		"import/no-unresolved": "off"
+	}
+}
```

Comment:
This turns off sorting rules for the SDK tests as some importing orders are important for the tests to succeed. There's also some mapping from `raiden-ts` imports to `src` going on which is easy in jest but not so trivial for eslint